src/templates/charm_policy.hcl

```diff
@@ -0,0 +1,66 @@
+# Allow management of policies starting with charm- prefix
+path "sys/policy/charm-*" {
+  capabilities = ["create", "read", "update", "delete"]
+}
```

Comment:
in `[_ensure_unit_credentials](https://github.com/canonical/vault-k8s-operator/blob/main/src/charm.py#L787)` we configure a policy that doesn't use this prefix.
Will that be a problem?

Reply:
which policy would that be? We only configure policies that start with charm [here](https://github.com/canonical/vault-k8s-operator/blob/058889f586c4e440a4cb502fa0d8af11ecb8eb85/src/charm.py#L426).